tdrs-frontend/reports/zap.conf

```diff
@@ -0,0 +1,4 @@
+90022	IGNORE	(Application Error Disclosure)
```

Comment:
@riatzukiza with exception to 10036, can we add in-line comments for each of these to describe why these have been marked as false positives? Alternatively, these can be added to #879 which includes adding this documentation as a task. cc: @abottoms-coder @amilash

Reply:
None of these show up in a scan on a production environment, these alerts only effect the test instance.

Reply:
I  90022 needs a little research.

Reply:
will tag this one as blocked pending more research on 90022.

Reply:
This route doesn't exist, its a 404. The alert indicates it thinks an internal error occured, this doesn't happen when you visit it on a production website @ADPennington

Reply:
thanks @riatzukiza. @abottoms-coder feel free to add the qasp tag when ready cc: @amilash